assets/vue/convos-connection-editor.vue

```diff
@@ -104,6 +108,7 @@ module.exports = {
       userinfo = userinfo.match(/[^:]/) ? userinfo + "@" : "";
       connection.user = this.user;
       connection.url = this.selectedProtocol + "://" + userinfo + this.server;
+      connection.on_connect_commands = this.on_connect.split(/\n/);
```

Comment:
I think in some browsers, there's `\n\r` instead of just `\n`. How about doing this:
```
this.onConnectCommands.split(/\n/).map(function(cmd) {
  return cmd.trim(); // trim removes \s\r\n
});
```